torch/csrc/jit/tensorexpr/operators/reduction.cpp

```diff
@@ -116,7 +117,7 @@ Tensor* computeMean(
   } else {
     // When dims argument is not specified, reduce over all dimensions
     // NOLINTNEXTLINE(clang-diagnostic-sign-compare)
-    for (int64_t idx = 0; idx < InputBuf.ndim(); idx++) {
+    for (const auto idx : c10::irange(InputBuf.ndim())) {
       mean_dims_expr.emplace_back(idx);
```

Comment:
Hmm, you'll need to explicitly typecast `idx` to one of the types defined in 
https://github.com/pytorch/pytorch/blob/d6e776d96110a3b0c7d4777b03e4c400ae9a46eb/c10/core/ScalarType.h#L125-L131